src/components/Filters/README.md

```diff
@@ -0,0 +1,481 @@
+---
+name: Filters
+category: Lists and tables
+keywords:
+  - filters
+  - filtering
+  - filter control
+  - resource list
+  - index
+  - list filter
+  - table
+---
+
+# Filters
+
+Filters is a composite component that filters the items of a list or table.
```

Comment:
I prefer being general because it is a bit more future-proof and leaves it open for creative/flexible implementations if, for example, a consumer has built their own kind of custom list.